db_connection.ts

```diff
@@ -1,21 +1,24 @@
-import { MongoClient, Db } from 'mongodb';
+import { MongoClient } from 'mongodb';
 
 import config from './config';
 import { Duty } from './schemas/duty.zschema';
 import { Soldier } from './schemas/soldier.zschema';
 
-let connection: Db;
+let client: MongoClient;
 
-const connect = async () => {
-  const client = await MongoClient.connect(config.URI_DB);
-  connection = client.db(config.NAME_DB);
+const startConnection = async () => {
+  client = await MongoClient.connect(config.URI_DB);
+};
+
+const closeConnection = async () => {
+  await client.close();
 };
 
 const getConnection = async () => {
```

Comment:
I don't like getConnection because it connects and has a sideEffect, but you can leave it